src/win/util.c

```diff
@@ -1156,6 +1156,40 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
   return 0;
 }
 
+int uv_getrusage_thread(uv_rusage_t *uv_rusage) {
+  FILETIME createTime, exitTime, kernelTime, userTime;
+  SYSTEMTIME kernelSystemTime, userSystemTime;
+  int ret;
+
+  ret = GetThreadTimes(GetCurrentThread(), &createTime, &exitTime, &kernelTime, &userTime);
```

Comment:
Style: long line (it should be 80 chars max).
While you're here, could you wrap also the same line in `uv_getrusage()`? Thanks!